_includes/main_menu.html

```diff
@@ -39,7 +29,7 @@
       <a href="{{ site.baseurl }}/docs">Docs</a>
     </li>
 
-    <li class="main-menu-item {% if current[1] == 'resources' or current[1] == 'features' %}active{% endif %}">
+    <li class="main-menu-item {% if current[1] == 'resources' or current[1] == 'features' or current[1] == 'hub' %}active{% endif %}">
```

Comment:
I think this can be cleaned up now that we're checking for 3 links. You can create an array for the three page names and then check if the array contains `current[1]`. Check out contains [here](https://shopify.github.io/liquid/basics/operators/) and how to create an array [here](https://shopify.github.io/liquid/filters/split/).